theano/sandbox/cuda/opt.py

```diff
@@ -1128,11 +1128,9 @@ def local_gpu_advanced_incsubtensor1(node):
             compute_capability = device_properties(active_device_no)['major']
             if (compute_capability < 2 or y.ndim != 2 or x.ndim != 2):
 
-                gpu_op = GpuAdvancedIncSubtensor1(
-                    set_instead_of_inc=set_instead_of_inc)
+                gpu_op = tensor.AdvancedIncSubtensor1(**node.op._props_dict())
```

Comment:
I remember why. Since we're passing the `_props_dict` and this class( `GpuAdvancedIncSubtensor1`) as well it's parents don't have `__init__` or `__props__`, we'd get errors when we do this.